dev/test/bundle.ts

```diff
@@ -37,17 +38,17 @@ export function verifyMetadata(
   expectEmptyContent = false
 ): void {
   if (!expectEmptyContent) {
-    expect(meta.totalBytes).greaterThan(0);
+    expect(parseInt(meta.totalBytes!.toString())).greaterThan(0);
```

Comment:
I think you can probably emit toString() here (totalBytes should be a string).

Reply:
It's there to silent the type error, it can be number or string.